pycbc/injcutter.py

```diff
@@ -0,0 +1,318 @@
+# Copyright (C) 2013 Ian Harry
```

Comment:
I think this should be paired with the InjectionSet class. Can these both be placed into a pycbc.injection package?